src/oaklib/io/streaming_fhir_writer.py

```diff
@@ -24,10 +24,31 @@ def emit_multiple(self, entities: Iterable[CURIE], **kwargs):
             g = oi.extract_graph(list(entities), include_metadata=True)
             gd = GraphDocument(graphs=[g])
             logging.info(f"Converting {len(g.nodes)} nodes to OBO")
-            converter = OboGraphToFHIRConverter()
+            converter = OboGraphToFhirJsonConverter()
             converter.curie_converter = oi.converter
             code_system = converter.convert(gd)
             logging.info(f"Writing {len(code_system.concept)} Concepts")
+            # TODO: Should not this call OboGraphToFhirJsonConverter.dump()?
             self.file.write(json_dumper.dumps(code_system))
```

Comment:
@cmungall In `StreamingFhirJsonWriter` I see that it does a dump, but not the way I would expect? See comment:
`            # TODO: Should not this call OboGraphToFhirJsonConverter.dump()?`

Reply:
I think you wrote this code, and I'm thinking this might be a bug. I don't know how `StreamingFhirJsonWriter` is used.